### DISCUSSION TITLE
Newbie question on Strategy for Multi-Tenancy

### DISCUSSION THREAD

**Discussion by 274188A** [Finbuckle/Finbuckle.MultiTenant#649] [Q&A] [answered]
My question is best explained in a small scenario:

If a page has say a drop-down containing "days-of-the-week"  - it's intended to be used across all tenants - but the data should only editable by some sort of 'host' (not the tenants themselves).

My definition of a a 'host' is that the host exists only in order to manage tenants and permissions to 'shared' data like the 'days-of-the-week'

Is there such a strategy 'out-of-the-box'? - 
Thanks

**Comment by AndrewTriesToCode** [comment, marked as answer]
Hi, I'm assuming the drop down option choices are stored in a database that is otherwise configured for multitenant? It's reasonable that you would just not mark that table multitenant and the library will basically ignore it when doing its internal logic.

Let me know if I'm understanding correctly.

**Reply by 274188A** [reply]
i see - too easy - thanks!
